src/main/java/uk/gov/hmcts/ccd/domain/model/draft/DraftResponse.java

```diff
@@ -59,6 +59,13 @@ public void setUpdated(LocalDateTime updated) {
         this.updated = updated;
     }
 
+    public DraftResponse() {
```

Comment:
Is empty default constructor needed?

Reply:
Probably not, but just want to get `global_search` updated with `master` - we can review in detail when there is an excercise to merge `global_search` onto `master` branch

Reply:
Sounds good!